taxi/views.py

```diff
@@ -87,3 +85,20 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
+
+
+class DriverCreateView(LoginRequiredMixin, generic.CreateView):
+    model = Driver
+    fields = "__all__"
```

Comment:
The `DriverCreateView` should use `UserCreationForm` instead of `forms.ModelForm` to create a user. This aligns with the checklist requirement: "Use `UserCreationForm` while creating user."